pls/rootchain_manager.go

```diff
@@ -334,14 +328,20 @@ func (rcm *RootChainManager) runSubmitter() {
 
 func (rcm *RootChainManager) runHandlers() {
 	for {
+		currentBlockNumber := rcm.blockchain.CurrentBlock().NumberU64()
```

Comment:
`currentBlockNumber` is current block of *PLASMA* block. it should be the root chain block number

Reply:
https://github.com/Onther-Tech/plasma-evm/pull/38/commits/9836783bca3d3ac540e8fe434fcc2616b246a06d

Reply:
use `currentBlock = e.Raw.BlockNumber`